OpenRA.Game/Widgets/Widget.cs

```diff
@@ -626,7 +626,11 @@ public class ContainerWidget : Widget
 
 		public ContainerWidget() { IgnoreMouseOver = true; }
 		public ContainerWidget(ContainerWidget other)
-			: base(other) { IgnoreMouseOver = true; }
+			: base(other)
+		{
+			ClickThrough = other.ClickThrough;
+			IgnoreMouseOver = true;
+		}
```

Comment:
this feels like it's going to produce countless regressions

Reply:
I think this was the only place where `ClickThrough` was used on a `ContainerWidget`.

Reply:
To observe this change you need to have set `ClickThrough: false` *and* clone it because it's a template for something. That's not going to be a common case.

I could only find a handful of locations with this combo - and it feels like a bugfix for all of them to me.

https://github.com/OpenRA/OpenRA/blob/a44e956ed93699f2f7a0bfbb1f7bb48b0c5ef4b3/mods/cnc/chrome/ingame.yaml#L1207-L1212

https://github.com/OpenRA/OpenRA/blob/a44e956ed93699f2f7a0bfbb1f7bb48b0c5ef4b3/mods/d2k/chrome/ingame-player.yaml#L477-L482

https://github.com/OpenRA/OpenRA/blob/a44e956ed93699f2f7a0bfbb1f7bb48b0c5ef4b3/mods/ra/chrome/ingame-player.yaml#L434-L441

https://github.com/OpenRA/OpenRA/blob/a44e956ed93699f2f7a0bfbb1f7bb48b0c5ef4b3/mods/ts/chrome/ingame-player.yaml#L477-L483